templates/blog.amp.json

```diff
@@ -18,7 +18,7 @@
         "width": "100",
         "height": "61.3"
       },
-      "name": "The Blog",
+      "name": "The Blog logo",
```

Comment:
Could you provide proper text for this then? The Templates and things usually have random logos.